### PR TITLE
[cases](regression-test) Add backup & restore test case of dup table

### PR DESCRIPTION
**Pull request by Bears0haunt**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Bears0haunt** [comment]
run buildall

**Comment by Bears0haunt** [comment]
run buildall

File: regression-test/suites/backup_restore/test_backup_restore_dup_without_default_keys.groovy [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    sql "TRUNCATE TABLE ${dbName}.${tableName}"
+
+    sql """
+        RESTORE SNAPSHOT ${dbName}.${snapshotName}
+        FROM `${repoName}`
+        ON (`${tableName}`)
+        PROPERTIES
+        (
+            "backup_timestamp" = "${snapshot}",
+            "replication_num" = "1",
```
Comment (w41ter) [inline regression-test/suites/backup_restore/test_backup_restore_dup_without_default_keys.groovy:72]:
```suggestion
            "replication_num" = "1"
```

File: regression-test/suites/backup_restore/test_backup_restore_dup_without_default_keys.groovy [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_backup_restore_dup_without_default_keys", "backup_restore"){
+    String repoName = "test_backup_restore_dup_without_default_keys"
```
Comment (w41ter) [inline regression-test/suites/backup_restore/test_backup_restore_dup_without_default_keys.groovy:19]:
```suggestion
suite("test_backup_restore_dup_without_default_keys", "backup_restore") {
    String repoName = "test_backup_restore_dup_without_default_keys_repo"
```

**Review by w41ter — CHANGES_REQUESTED** [review]
Reset LGTM.

**Comment by Bears0haunt** [comment]
run buildall

**Comment by Bears0haunt** [comment]
run buildall

**Review by w41ter — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**